### PR TITLE
🌐 Lot of enhancements on Italian localization

### PR DESCRIPTION
**Pull request by M0Rf30**
Hereby I submit a lot of enhancements in particular:
* Menu entries
* Accented vowels (such a pity that accents are not supported in textdat strings, so I'm forced to replace `à` with `a'`)
* Tristram experience (Shops in particular)

Another issue that I encountered is related to items qualities. An example is more than thousand words

In English I say `Silver Sword` and the piece of code that concatenate strings displays  `Silver Sword`.
In Italian I say `Sword` (Spada) `of` (di) `Silver` (Argento) and strings are concatenated as `Argento Spada` (a sort of `Sword Silver`).
That's because adjective and nouns are naturally swapped between Italian and English. Use of `of` could be avoided playing on grammar, but the swapping make the text not so delightful

At a first glance seems to be related at least to modifiers that describe the kind of material (brass, silver, obsidian and so on)

### CODE REVIEW

**Comment by AJenbo** [comment]
> Accented vowels (such a pity that accents are not supported in textdat strings, so I'm forced to replace `à` with `a'`)

They are, the font is not included yet though. But you can install it manually from here if you want to test things out https://github.com/diasurgical/devilution/issues/32#issuecomment-402322326. So please don't use `a'`, use the propper `à` and be patient :)

> In Italian I say Sword (Spada) of (di) Silver (Argento)

This will become possible in the not-to-distant future. There will be a translation option for  `{Prefix} {ItemName}` which allows you to translate it as `{ItemName} di {Prefix}` to change how item names are constructed.

**Comment by M0Rf30** [comment]
Ok I'll fix accented vowels and repush :smile:

**Comment by M0Rf30** [comment]
> This will become possible in the not-to-distant future. There will be a translation option for {Prefix} {ItemName} which allows you to translate it as {ItemName} di {Prefix} to change how item names are constructed. 

Amazing job man

>They are, the font is not included yet though. But you can install it manually from here if you want to test things out diasurgical/devilution#32 (comment). So please don't use a', use the propper à and be patient :)

Done. And yes I'll patiently wait :smile: although in the yet opened discussion I didn't catch the making of a new shiny font. So great

**Comment by hiperiondev** [comment]
> Another issue that I encountered is related to items qualities. An example is more than thousand words
> 
> In English I say `Silver Sword` and the piece of code that concatenate strings displays `Silver Sword`.
> In Italian I say `Sword` (Spada) `of` (di) `Silver` (Argento) and strings are concatenated as `Argento Spada` (a sort of `Sword Silver`).
> That's because adjective and nouns are naturally swapped between Italian and English. Use of `of` could be avoided playing on grammar, but the swapping make the text not so delightful
> 
> At a first glance seems to be related at least to modifiers that describe the kind of material (brass, silver, obsidian and so on)

Exactly the same applies to the Spanish.
Another problem is the Saxon genitive, which does not exist in Latin languages.

**Comment by AJenbo** [comment]
could you give an example, and how you would do it if there where no restrictions?

**Comment by M0Rf30** [comment]
@AJenbo `msgid "%s Scroll"` has some troubles:
`format specifications in 'msgid_plural' and 'msgstr[0]' for argument 1 are not the same`

**Comment by AJenbo** [comment]
> @AJenbo `msgid "%s Scroll"` has some troubles:

Thanks that has now been corrected in src

File: Translations/it.po [hunk 1]
```diff
@@ -999,106 +983,95 @@ msgstr "Attacca Eroi"
...
+msgstr "Tasto: 's'"
 
-#: Source/control.cpp:1069
+#: Source/control.cpp:1057
 #, c-format
-msgid "%i Charges"
-msgstr "%i Cariche"
+msgid "%s Scroll"
+msgid_plural "%i Scrolls"
+msgstr[0] "%s Pergamena"
```
Comment (AJenbo) [inline Translations/it.po:1000]:
```suggestion
msgid "%i Scroll"
msgid_plural "%i Scrolls"
msgstr[0] "%i Pergamena"
```

**Pull request merged**